sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -102,7 +102,7 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
     }
     renderer.onStart()
     pluginRegistry.onStart()
-    if (!mapboxMap.isStyleLoadInited) {
+    if (!mapboxMap.isStyleLoadInitiated) {
```

Comment:
It's a bit strange we see it here, is this PR correctly rebased?

Reply:
We have an internal ticket tracking this rename #352, JFDI 
Updated changelog entry in description + added breaking change label